EnglishFilter/sections/foreign.txt

```diff
@@ -823,6 +823,8 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/42425
 javporn.best#@#.afs_ads
 ||javporn.best/asset/default/player/plugins/vast-*.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/42481
+@@||cloudflare.com/ajax/libs/fingerprintjs2/*/fingerprint2.min.js$domain=nfmovies.com
```

Comment:
As I see, this request is blocked by `AdGuard Tracking Protection filter`, so I guess that `!#safari_cb_affinity(privacy)` may be needed, or not?
```
!#safari_cb_affinity(privacy)
@@||cloudflare.com/ajax/libs/fingerprintjs2/*/fingerprint2.min.js$domain=nfmovies.com
!#safari_cb_affinity
```